ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -22,23 +22,35 @@
     (v/validate-concept-metadata concept)
     concept))
 
+(defn- subscriber-collection-permission-error
+  [subscriber-id concept-id]
+  (errors/throw-service-error
+   :unauthorized
+   (format "Subscriber-id [%s] does not have permission to view collection [%s]."
```

Comment:
Can we change the error message to something like 
`(format "Collection with concept id [%s] does not exist or subscriber-id [%s] does not have permission to view the collection." concept-id subscriber-id)`?

Reply:
sure